client/src/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import {
+  Container,
+  Header,
+  List,
+  Grid,
+  Segment,
+  Icon
+} from "semantic-ui-react";
+
+export const Footer = () => (
+  <Segment inverted vertical style={{ padding: "1em 0em", marginTop: "6em" }}>
```

Comment:
I think the padding can be removed here? Also marginTop can be set to 5em instead, otherwise there is always a scrollbar on the right of the page.

Reply:
it is done